recipe/meta.yaml

```diff
@@ -26,6 +29,10 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}  # [cuda_compiler_version not in (undefined, 'None')]
+    {% if cuda_major >= 12 %}
+    - cuda-driver-dev
+    - cuda-nvrtc-dev
+    {% endif %}
```

Comment:
Think we can drop these from `build`. Having them in `host` should be sufficient

```suggestion
```